src/Set.ts

```diff
@@ -123,7 +123,7 @@ class Set {
       assertions
     );
     this.system = new SystemAPI(ethersProvider, config.controllerAddress);
-    this.trade = new TradeAPI(ethersProvider, config.tradeModuleAddress);
+    this.trade = new TradeAPI(ethersProvider, config.tradeModuleAddress, config.zeroExApiKey);
```

Comment:
I don't know if people can generate their own API keys for 0x. We may have to supply it for it to be used by anyone else

Reply:
> The 0x API is configured with a list of API keys which are permitted to access RFQ-T liquidity. For the instance at api.0x.org, the 0x team is maintaining a list of trusted integrators

@asoong based on the statement from their docs it's not possible to request one. If I understand correctly it will still works but I just may not get the best deal available, right?